lib/web/mfa.go

```diff
@@ -180,6 +180,8 @@ type createAuthenticateChallengeRequest struct {
 // createAuthenticateChallengeHandle creates and returns MFA authentication challenges for the user in context (logged in user).
 // Used when users need to re-authenticate their second factors.
 func (h *Handler) createAuthenticateChallengeHandle(w http.ResponseWriter, r *http.Request, p httprouter.Params, c *SessionContext) (interface{}, error) {
+	ctx := r.Context()
```

Comment:
why not use `r.Context()`?

Reply:
Just because we're using it several times below, I found it more dextrous to rename it.